### DISCUSSION TITLE
Request for Information on ROS 2 Compatibility for industrial_msgs

### DISCUSSION THREAD

**Discussion by Saketh7256** [Yaskawa-Global/motoros2_client_interface_dependencies#9] [Q&A] [answered]
Hi there,

I'm working on a project with ROS 2 Humble and I'm trying to use the industrial_msgs package. I couldn’t find a ROS 2 version in the Ubuntu repos or ROS 2 package indexes.

Do you know if industrial_msgs is available for ROS 2 ? Also, if there’s another package or branch that supports ROS 2, could you point me to it?

Thanks for your help!

**Comment by ted-miller** [comment, marked as answer]
Hi Saketh7256

The messages are available on this branch:
https://github.com/ros-industrial/industrial_core/tree/ros2_msgs_only

All the required dependencies are in [this file](https://github.com/Yaskawa-Global/motoros2_client_interface_dependencies/blob/master/source_deps.repos). The procedure on the README for this repo lists the steps to obtain the required packages using that file.
